contracts/adapters/FlexibleLeverageStrategyAdapter.sol

```diff
@@ -671,9 +682,11 @@ contract FlexibleLeverageStrategyAdapter is BaseAdapter {
     function _createActionInfo() internal view returns(ActionInfo memory) {
         ActionInfo memory rebalanceInfo;
 
-        // IMPORTANT: Compound oracle returns prices adjusted for decimals. USDC is 6 decimals so $1 * 10^18 * 10^18 / 10^6 = 10^30
-        rebalanceInfo.collateralPrice = strategy.priceOracle.getUnderlyingPrice(address(strategy.targetCollateralCToken));
-        rebalanceInfo.borrowPrice = strategy.priceOracle.getUnderlyingPrice(address(strategy.targetBorrowCToken));
+        // Calculate prices from chainlink, adjusting for decimals.
```

Comment:
Can you make this comment a bit more robust explaining the exact adjustment being made? What decimal does Chainlink return and how are we adjusting it. Still don't really understand how these adjustments and the price fit together. Does this work for an asset with 4 decimals?

Reply:
Are we trying to return an 18 decimal price here? Or some other adjusted price?